### PR TITLE
Feature/#102 upgrade the version of node

### PR DESCRIPTION
**Pull request by ggrg**

### CODE REVIEW

**Review by NicoDuvenage — CHANGES_REQUESTED** [review]
👎 Please fill in you tasks, tick them off and also do the acceptance criteria tick-boxes in the story on the board.

**Review by NicoDuvenage — APPROVED** [review]
👍

**Review by rmothilal — CHANGES_REQUESTED** [review]
Hi Georgi, you need to make changes to the file:
/.circleci/config.yml
you need to update on line 7 to:
docker:
    - image: circleci/node:8.9.4

from 
 docker:
    - image: dwolla/alpine-node-make

**Review by mdebarros — DISMISSED** [review]
You also need make changes to the following docker files:
- Dockerfile
- test.Dockerfile

And update the following:
`FROM mhart/alpine-node:6.5.0` to the correct node version.

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ defaults_working_directory: &defaults_working_directory
 
 defaults_docker_node: &defaults_docker_node
   docker:
-    - image: dwolla/alpine-node-make
+    - image: circleci/node:8.9.4
```
Comment (rmothilal) [inline .circleci/config.yml]: Change this to mhart/alpine-node:8.9.4

Comment (ggrg) [reply .circleci/config.yml]: ok

File: .circleci/config.yml [hunk 2]
```diff
@@ -123,19 +123,37 @@ jobs:
           name: Link NPM sodium
           command: npm link sodium
       - run:
```
Comment (rmothilal) [inline .circleci/config.yml]: the order here is incorrect you cannot link sodium before it is installed. move line 122 to 124 below the install argon command

File: .circleci/config.yml [hunk 3]
```diff
@@ -123,19 +123,37 @@ jobs:
...
+          command: apk add --no-cache -t build-dependencies make gcc g python libtool autoconf automake
+      - run:
+          name: Access npm folder as root
+          command: cd $(npm root -g)/npm
+      - run:
+          name: Install fs-extra
+          command: npm install fs-extra
+      - run:
+          name: Edit something
+          command: sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js
```
Comment (rmothilal) [inline .circleci/config.yml]: i told you to delete the above lines:

     - run:
          name: Install fs-extra
          command: npm install fs-extra
      - run:
          name: Edit something
          command: sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js

File: .circleci/config.yml (hunk 2 above)
Comment (ggrg) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 3 above)
Comment (ggrg) [reply .circleci/config.yml]: done

File: .circleci/config.yml [hunk 4]
```diff
@@ -119,23 +119,35 @@ jobs:
       - run:
           name: Install interledgerjs/five-bells-ledger-api-tests
           command: npm install github:interledgerjs/five-bells-ledger-api-tests
+      - run:
+          name: Build dependencies
+          command: apk add --no-cache -t build-dependencies make gcc g python libtool autoconf automake
```
Comment (rmothilal) [inline .circleci/config.yml]: this is supposed to be: 
apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake

you didnt put "g++"

**Review by rmothilal — APPROVED** [review]

**Pull request merged**